api/v1beta1/openstackcontrolplane_types.go

```diff
@@ -28,6 +28,11 @@ type OpenStackControlPlaneSpec struct {
 	OpenStackClientImageURL string `json:"openStackClientImageURL"`
 	// PasswordSecret used to e.g specify root pwd
 	PasswordSecret string `json:"passwordSecret,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default={ctlplane,external}
+	// OpenStackClientNetworks the name(s) of the OpenStackClientNetworks used to attach the openstackclient to
+	OpenStackClientNetworks []string `json:"openStackClientNetworks"`
```

Comment:
I wonder if unset if we just always made these the same as the 'controlplane' vmset networks?

Reply:
yes we could do that if we want the openstackclient to always be attached to all the networks